bodhi/server/consumers/masher.py

```diff
@@ -999,57 +1112,65 @@ def _get_master_repomd_url(self, arch):
         return master_repomd % (self.release.version, arch)
 
 
-class MashThread(threading.Thread):
-    """
-    A Thread that performs the subprocess call to mash.
-
-    Attributes:
-        log (logging.Logger): A logger for the thread to use.
-        mash_cmd (basestring): The command to run, including arguments.
-        name (basestring): The tag being mashed. This is set so that the thread name will appear in
-            the logs.
-        success (bool): True if the subprocess finished with exit code 0, False if the process is
-            still running or exited with a non-0 exit code.
-        tag (basestring): The tag being mashed.
-    """
+class RPMMasherThread(MasherThread):
+    ctype = ContentType.rpm
+    pungi_template_config_key = 'pungiconf.rpm'
+
+    def copy_additional_pungi_files(self, pungi_conf_dir, template_env):
+        self.git_clone('https://pagure.io/fedora-comps.git',
+                       'master',
+                       os.path.join(pungi_conf_dir, 'comps'))
+        out, err, ret = util.cmd('make', cwd=os.path.join(pungi_conf_dir, 'comps'))
+        if ret != 0:
+            self.log.error('out: %s', out)
+            self.log.error('Err: %s', err)
+            raise Exception('Unable to make comps')
+        shutil.copy('/etc/bodhi/variants-fedora.xml', pungi_conf_dir)
```

Comment:
Will variants be specific to the release, or do all Fedora releases have the same variants document?

Reply:
Let's make it just a template.